src/ml/neural_net/style_transfer/mps_style_transfer_vgg_16_block_1_node.m

```diff
@@ -0,0 +1,80 @@
+                                                  label:descriptor.conv1.label
+                                          updateWeights:descriptor.conv1.updateWeights
+                                                 device:dev
+                                               cmdQueue:cmdQ];
+
+    _relu1 = [MPSCNNNeuronReLUNNode nodeWithSource: [_conv1 resultImage]];
+
+    _conv2 = [MPSCNNConvolutionNode createConvolutional:[_relu1 resultImage]
+                                            kernelWidth:descriptor.conv2.kernelWidth
+                                           kernelHeight:descriptor.conv2.kernelHeight
+                                   inputFeatureChannels:descriptor.conv2.inputFeatureChannels
+                                  outputFeatureChannels:descriptor.conv2.outputFeatureChannels
+                                            strideWidth:descriptor.conv2.strideWidth
+                                           strideHeight:descriptor.conv2.strideHeight
+                                           paddingWidth:descriptor.conv2.paddingWidth
+                                          paddingHeight:descriptor.conv2.paddingHeight
+                                                weights:weights[[NSString stringWithFormat:@"%@/%@", name, @"block_1_conv_2_weights"]]
+                                                 biases:weights[[NSString stringWithFormat:@"%@/%@", name, @"block_1_conv_2_biases"]]
+                                                  label:descriptor.conv2.label
+                                          updateWeights:descriptor.conv2.updateWeights
+                                                 device:dev
+                                               cmdQueue:cmdQ];
+
+    _relu2 = [MPSCNNNeuronReLUNNode nodeWithSource: [_conv2 resultImage]];
+
+    _pooling = [[MPSCNNPoolingAverageNode alloc] initWithSource:[_relu2 resultImage]
+                                                    kernelWidth:descriptor.pooling.kernelSize
+                                                   kernelHeight:descriptor.pooling.kernelSize 
+                                                strideInPixelsX:descriptor.pooling.strideSize
+                                                strideInPixelsY:descriptor.pooling.strideSize];
+
+    _output = [_pooling resultImage];
+    _features = [_relu2 resultImage];
+  }
+  return self;
+}
+
+- (MPSNNImageNode *) backwardPass:(MPSNNImageNode *) inputNode {
+  MPSNNGradientFilterNode* poolingGrad = [_pooling gradientFilterWithSource: inputNode];
+  MPSNNGradientFilterNode* relu2Grad = [_relu2 gradientFilterWithSource: [poolingGrad resultImage]];
+  MPSNNGradientFilterNode* conv2Grad = [_conv2 gradientFilterWithSource: [relu2Grad resultImage]];
+  MPSNNGradientFilterNode* relu1Grad = [_relu1 gradientFilterWithSource: [conv2Grad resultImage]];
+  MPSNNGradientFilterNode* conv1Grad = [_conv1 gradientFilterWithSource: [relu1Grad resultImage]];
```

Comment:
Mega-nit: FYI, since `resultImage` is defined as a property (https://developer.apple.com/documentation/metalperformanceshaders/mpsnnfilternode/2866492-resultimage?language=objc), it's kosher to do `relu1Grad.resultImage` instead of `[relu1Grad resultImage]`. Honestly, I wouldn't bother changing this in this PR, but for future reference, I find the property syntax more readable. Fewer nested delimiters to parse.

Reply:
Perfect thank you!